.github/workflows/validate-release.yml

```
@@ -20,24 +28,48 @@ on:
+      SERVER_START_DELAY: 3
+      # License Patterns (ASF Category X - Prohibited)
+      CATEGORY_X: '\bGPL|\bLGPL|Sleepycat License|BSD-4-Clause|\bBCL\b|JSR-275|Amazon Software License|\bRSAL\b|\bQPL\b|\bSSPL|\bCPOL|\bNPL1|Creative Commons Non-Commercial|JSON\.org'
+      # License Patterns (ASF Category B - Must be documented)
+      CATEGORY_B: '\bCDDL1|\bCPL|\bEPL|\bIPL|\bMPL|\bSPL|OSL-3.0|UnRAR License|Erlang Public License|\bOFL\b|Ubuntu Font License Version 1.0|IPA Font License Agreement v1.0|EPL2.0|CC-BY'
     steps:
       - name: Checkout source
         uses: actions/checkout@v4
-      - name: Install JDK ${{ matrix.java_version }}
+
+      - name: Install JDK ${{ env.JAVA_VERSION }}
         uses: actions/setup-java@v3
```

actionlint (1.7.9)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)